extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerRecorder.java

```diff
@@ -90,8 +91,11 @@ private static PolicyEnforcer createPolicyEnforcer(OidcTenantConfig oidcConfig,
         adapterConfig.setConnectionPoolSize(keycloakPolicyEnforcerConfig.connectionPoolSize);
 
         if (oidcConfig.proxy.host.isPresent()) {
-            adapterConfig.setProxyUrl(oidcConfig.proxy.host.get() + ":"
-                    + oidcConfig.proxy.port);
+            String host = oidcConfig.proxy.host.get();
+            if (!host.startsWith("http")) {
```

Comment:
I know I'm being a bit pedantic here but I would rather have the test do:
```
if (!host.startsWith("http://") && !host.startsWith("https://")) {
```
Because we could imagine someone having a host that is `httpfoobar.com`.

Reply:
Hey @gsmet No problems at all, it is worth avoiding any issues like this one, thanks for spotting it